src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.Sql.tests.ps1

```diff
@@ -103,6 +103,22 @@ InModuleScope Arcus.Scripting.Sql {
             } catch {
                 # We don't care if an exception is thrown; we just want to 'activate' the Azure SQL database.
             }
+
+            $tableNames = @()
+            try {
+                $tableNames = Run-AzSqlQuery $params "SELECT TABLE_NAME FROM INFORMATION_SCHEMA.TABLES"
+            } catch {
+                Write-Warning "Could not select any SQL tables to drop during test setup"
```

Comment:
Do we want to stop processing if the retrieval of tables fails? If we continue if this fails we execute tests based on already existing tables and the tests might fail.
In that case this should be a `throw` right?

Reply:
Yeah, this gave problems on Linux images. Still not sure why. I guess, since we run this on both environments, at least one will succeed. But didn't got the time yet to look into why it sometimes fails.

Reply:
But if it fails, do we want to only write a warning and continue with the script?

Reply:
I'll try to look into this. It's an insurance policy, these changes. The tests itself should handle the teardown. It's a back-up of a back-up, really. But still valid 😉 . I'll try to investigate in this, why it fails.

Reply:
Failure was a transient Azure SQL failure, so would need to retry that which made the test succeed.